draco/core/task.py

```diff
@@ -262,6 +262,9 @@ class SingleTask(MPILoggedTask, pipeline.BasicContMixin):
 
     output_root = config.Property(default="", proptype=str)
     output_name = config.Property(default="{output_root}{tag}.h5", proptype=str)
+    output_format = config.file_format()
+    output_compression = config.Property(default=None, proptype=str)
+    output_compression_opts = config.Property(default=None)
```

Comment:
I can't remember if these are just inherited from `BasicContMixin`, so check that...

Also, similar to my comments in `caput.pipeline` it's not clear if we should allow these to be specified *per dataset*. Also is this the right format for these arguments if we do that?